utils/linkage.py

```diff
@@ -309,7 +309,12 @@ def get_likely_name(first_name: str, last_name: str, full_name: str) -> str:
     >>> get_likely_name("Jane","","Doe, Jane, Elisabeth")
     'Jane Elisabeth Doe'
     """
-    # first ensure clean input by deleting spaces:
+    # first, convert any Nans to empty strings ''
+    first_name, last_name, full_name = [
+        "" if x is np.NAN else x for x in [first_name, last_name, full_name]
```

Comment:
small thing but I think you want to do this: np.isnan(x)

Reply:
I am not sure if that would work, bc np.isnan(x) returns an error for when x is not np.NaN. However, leaving it as is seems to be working for both nan and non nan values. 

![x1](https://github.com/dsi-clinic/2024-winter-climate-cabinet-campaign-finance-tracker/assets/72175018/870c8b68-149a-4296-aecf-f7b8c51f284d)
![x2](https://github.com/dsi-clinic/2024-winter-climate-cabinet-campaign-finance-tracker/assets/72175018/ac2e268e-d669-4855-b07d-37ba2d50df67)

lmk if you can access the reference images, but just in case they don't load:
`first_name, last_name, full_name = np.NaN, "dfd","dadf dfd"
first_name, last_name, full_name = ["" if x is np.nan else x for x in [first_name, last_name, full_name]]
first_name, last_name, full_name` yields ('', 'dfd', 'dadf dfd')
 
whereas 
`first_name, last_name, full_name = ["" if x is np.isnan(x) else x for x in [first_name, last_name, full_name]]`
yields 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [19], in <cell line: 2>()
      1 first_name, last_name, full_name = np.NaN, "dfd","dadf dfd"
----> 2 first_name, last_name, full_name = ["" if x is np.isnan(x) else x for x in [first_name, last_name, full_name]]
      3 first_name, last_name, full_name

Input In [19], in <listcomp>(.0)
      1 first_name, last_name, full_name = np.NaN, "dfd","dadf dfd"
----> 2 first_name, last_name, full_name = ["" if x is np.isnan(x) else x for x in [first_name, last_name, full_name]]
      3 first_name, last_name, full_name

TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''